src/fios.cpp

```diff
@@ -128,16 +127,11 @@ const FiosItem *FileList::FindItem(const std::string_view file)
 }
 
 /**
- * Get descriptive texts. Returns the path and free space
- * left on the device
- * @param path string describing the path
- * @param total_free total free space in megabytes, optional (can be nullptr)
- * @return StringID describing the path (free space or failure)
+ * Get the current path/working directory.
  */
-StringID FiosGetDescText(const char **path, uint64 *total_free)
+std::string FiosGetCurrentPath()
```

Comment:
not a `string_view`?

Reply:
No; it gets passed to `FiosGetDiskFreeSpace` which in its turn calls C-style OS APIs which want `'\0'`-terminated strings, which `std::string_view`s are not.